src/last10.c

```diff
@@ -54,18 +52,9 @@ int last10(void) {
 	}
     }
 
```

Comment:
From the description in the header we want to return -1 if less than 10 QSOs are found on the band. So should we better check for 'qsocount <10' instead of 'index <0'?

Reply:
You are right. I misread the code. Keep it as is.